### ISSUE TITLE
can not complete windows 10 connection wizard 

### ISSUE THREAD

**Issue by rtprio** [nextcloud/desktop#1280]
There seems to be a 303 response and the process seems to stop there. 
The Android and web ui appear to work fine. 

### Expected behaviour
connection wizard should complete 

### Actual behaviour
connection wizard hangs after pressing 'grant access'

### Steps to reproduce
1.  Follow the wizard
2.  enter correct username, password
3. click grant

### Client configuration
Client version:   Version 2.5.2git (build 20190319). 

Operating system:  Windows 10.0.17763 Build 17763

OS language:

Client package (From Nextcloud or distro) (Linux only):

Installation path of client: default 

### Server configuration

Operating system: Freebsd 13

Web server: nginx

Database: mysql : 5.6.43

PHP version: PHP 7.3.3

Nextcloud version: 16.0.1.1

Storage backend (external storage): local, zfs 

### Logs

1. Client logfile: 

https://gist.github.com/rtprio/89b80e9ac0a3d043158557713757c5dd

2. Web server error log:
```
2019/05/29 17:46:37 [info] 614#100356: *100009 kevent() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 10.0.17.4, server: _, request: "POST /login/flow/apptoken HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "cloud.17es.net"
2019/05/29 17:46:37 [info] 614#100356: *100011 kevent() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 10.0.17.4, server: _, request: "POST /login/flow/apptoken HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "cloud.17es.net"
```
3. Server logfile: nextcloud log (data/nextcloud.log):
There is no logs for this event in this log.

4. server access log

```10.0.17.4 - - [29/May/2019:18:06:35 +0000] "GET /core/js/oc.js?v=d67fada3 HTTP/1.0" 200 5516 "-" "Mozilla/5.0 (Windows) mirall/2.5.2git (build 20190319) (Nextcloud)" "1.2.125.196"
10.0.17.4 - - [29/May/2019:18:06:39 +0000] "GET /cron.php HTTP/1.0" 200 20 "-" "Mozilla/5.0 (Windows) mirall/2.5.2git (build 20190319) (Nextcloud)" "1.2.125.196"
10.0.17.4 - - [29/May/2019:18:06:40 +0000] "POST /login/flow HTTP/1.0" 303 0 "-" "Mozilla/5.0 (Windows) mirall/2.5.2git (build 20190319) (Nextcloud)" "1.2.125.196"
```

**Comment by rtprio** [comment]
Following up on my own issue, it would seem that the windows client doesn't support TLS1.2.

**Comment by misch7** [comment]
Please try our recent 2.6 release and re-open this ticket if the problem still persists with it:
https://nextcloud.com/install/#install-clients

In 2.6 we've implemented client SSL certificates and TLS 1.3 which might be the cause of your login issue.
